src/main/java/com/minecolonies/coremod/colony/IColony.java

```diff
@@ -1,9 +1,12 @@
 package com.minecolonies.coremod.colony;
 
 import com.minecolonies.api.colony.permissions.IPermissions;
+import net.minecraft.entity.EntityLivingBase;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.entity.EntityLivingBase'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)